src/main/java/core/basesyntax/FruitShopApp.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax;

```

Comment:
Interesting job. But I have a couple of questions for the author.
dao - data access object. What object are you looking at in the reader and writer?